integration/test_helpers/helpers.go

```diff
@@ -47,7 +43,7 @@ func RunCommand(cmd string, args ...string) *Session {
 
 	session, err := Start(command, GinkgoWriter, GinkgoWriter)
 	Expect(err).NotTo(HaveOccurred())
-	session.Wait()
+	session.Wait(10 * time.Second)
```

Comment:
Done. Also changed timeout to 3 seconds and this works for current integration test.